ci/deploy_credentials.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+if [[ -z "$PROJECT_ID" ]]; then
+  echo "PROJECT_ID not provided"
+  exit 1
+fi
+
+if [[ -z "$NOTIFY_API_KEY_FILE" ]]; then
+  echo "NOTIFY_API_KEY_FILE not provided"
```

Comment:
Does this need to be added to the pipeline params being passed?

``echo 'NOTIFY_API_KEY_FILE not provided'
NOTIFY_API_KEY_FILE not provided
``

Similar to how we do it for [Runner](https://github.com/ONSdigital/eq-pipelines/blob/master/development/staging/pipeline.yaml#L54-L55)

Reply:
It is being passed here https://github.com/ONSdigital/eq-pipelines/pull/266/files#diff-8e0a130fd413f1d4d34fa00643e7ab5fcfb55ed9a4c931195212091b8806b6deR98-R100

Reply:
Yep.. apologies. Just been across and seen you'd made this update !

Reply:
Nah, my bad, should have re-requested review here once the other one was ready, I guess you checked that other one out when I requested here, and before I pushed.